web/client/themes/default/less/dropdown-menu.less

```diff
@@ -92,6 +92,8 @@
 
 #mapstore-burger-menu {
     .dropdown-menu {
-        z-index: 1040;
+		z-index: 1040;
+		overflow-y: scroll;
+        max-height: 80vh;
```

Comment:
max-height 80vh works when mapstore cover the whole page.
I tested in the project and it seems 80vh is not enough.

![image](https://user-images.githubusercontent.com/19175505/106627093-b7427980-6578-11eb-90e7-6b4995131152.png)

could we try 70vh?
![image](https://user-images.githubusercontent.com/19175505/106627168-cde8d080-6578-11eb-99c8-c68db1f13a9f.png)
 
the solution proposed by @luorlandini is good,
I would like to highlight that this kind of fix depends on the structure of the specific project so probably we could apply the style in the custom css of the project (cc @tdipisa)